### DISCUSSION TITLE
How to update?

### DISCUSSION THREAD

**Discussion by franblade** [wlcrs/huawei_solar#104] [Setting up the integration] [answered]
Hi!

I installed this integration some months ago and it's working fine.

Now, I want to update without lose the actual configuration, how's the best option to do this? 

Thanks.

**Comment by tarmor1** [comment, marked as answer]
This is how I updated:
```

1. cd custom_components/
2. rm -r huawei_solar/
3. tar -xvzf 1.2.0b1.tar.gz
4. mv huawei_solar-1.2.0b1/ huawei_solar/
```
.. and [1.2.0 beta 1](https://github.com/wlcrs/huawei_solar/releases/tag/1.2.0b1) with sensors history all OK under HA 2022.12.0
